scripts/decredPoliteia.md

```diff
@@ -1,25 +1,29 @@
+To achieve this chain-anchoring, Politeia's off-chain storage system makes use of Git (a popular revision control system), and DecredTime (a custom server application for storing timestamps on the Decred blockchain.)
 
-Like reddit, content posted to Politeia must be manually reviewed by a human. This presents a problem common to all social media today: silent censorship. Politeia has a system in place to deal with censorship. 
+Git tracks, hashes, and stores changes to Decred's governance data as revisions to create a historic record of every change to the data stored on Politeia. These revisions are then anchored to the Decred blockchain through DecredTime to construct a time sequence of data revisions that is computationally impossible to recreate.
 
-It works like this:
+In order to enforce proposal quality standards and limit the amount of on-chain storage required, proposals submitted through Politeia must be manually reviewed by a human.
 
-Once a proposal is submitted, it is classified as ‘unvetted’ and the user who submitted the proposal receives a censorship token. After the proposal has been reviewed and deemed in adherence with proposal guidelines, it is considered ‘vetted’ and becomes available for public viewing. When a proposal becomes vetted, the corresponding censorship token is no longer valid. If a proposal never becomes vetted, the proposal submitter can use their censorship token to publicly prove their proposal was censored. By using censorship tokens, politeia creates a transparent censorship process. Administrators can be held publicly responsible for their actions when found to be acting inappropriately.
+In a system where humans must moderate content, censorship is likely to occur. However, content on Politeia cannot be silently censored thanks to a transparent review process made possible by tokens.
 
-Politeia is the ideal infrastructure for self-governance of a crypto-currency. To contribute to Decred’s governance, head to decred.org to download the software for your platform and buy decred on exchanges such as bittrex and poloniex.
+Once a stakeholder sumbmits a proposal on Politeia for review, they receive a censorship token. After their proposal has been reviewed and deemed in adherence with guidelines, it becomes available for public consideration. Once a proposal becomes public, it's corresponding censorship token becomes invalid. However, if a stakeholder's proposal is rejected, their censorship token remains valid and can be used publicly prove that their proposal was censored. The use of censorship tokens in this manner allows stakeholders to hold human administrators publicly responsible for their actions when found to be acting inappropriately.
 
-Decred, Decentralized Credits.
+Politeia is the ideal crytptographically verifiable and censorship resistant platform for self-governance of a crypto-currency. 
```

Comment:
Not sure if relevant in this context., but any serious project with open governance process could benefit from Pi, not only cryptocurrencies.